allennlp_runner.py

```diff
@@ -0,0 +1,32 @@
+import json
```

Comment:
This is a runner for the tuple embedder (can be used for pretty much any other AllenNLP task we have)